docs/tutorials/todo_list.md

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
I don't think we are aligned on this - can you remove from this PR and create a new ticket so we can discuss further?

Reply:
Sure, you're right. will remove it.